### PR TITLE
OSDOCS#11004: Part 2: Migration: Manage AWS on HCP

### PR DESCRIPTION
**Pull request by xenolinux**
This PR is a part of HCP content migration and modularization. The content has not been changed for this PR.
This content is only being moved from RHACM to OCP. This content has been SME and peer reviewed earlier. This migration PR contains the QE approval

Version(s): 4.17 and main

Issue: https://issues.redhat.com/browse/OSDOCS-11004

Link to docs preview: 
Whole assembly to review -- https://82143--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-manage/hcp-manage-aws.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/hcp-managed-aws-infra-separate.adoc [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+//
+// * hosted_control_planes/hcp-manage/hcp-manage-aws.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="hcp-managed-aws-infra-separate_{context}"]
+= Creating the {aws-short} infrastructure separately
+
+To create the {aws-first} infrastructure, you need to create a Virtual Private Cloud (VPC) and other resources for your cluster. You can use the {aws-short} console or an infrastructure automation and provisioning tool. For instructions to use the {aws-short} console, see link:https://docs.aws.amazon.com/vpc/latest/userguide/create-vpc.html#create-vpc-and-other-resources[Create a VPC plus other VPC resources] in the {aws-short} Documentation.
+
+The VPC must include private and public subnets and resources for external access, such as an NAT gateway and an internet gateway. In addition to the VPC, you need a private hosted zone for the ingress of your cluster. If you are creating clusters that use PrivateLink (`Private` or `PublicAndPrivate` access modes), you need an additional hosted zone for PrivateLink.
```
Comment (lpettyjo) [inline modules/hcp-managed-aws-infra-separate.adoc:11]: Spell out NAT acronym on first usage.

**Review by lpettyjo — COMMENTED** [review]
Otherwise, LGTM!

File: modules/hcp-managed-aws-infra-separate.adoc (hunk 1 above)
Comment (xenolinux) [reply modules/hcp-managed-aws-infra-separate.adoc]: Addressed

**Pull request merged**